README.md

```diff
@@ -1,5 +1,7 @@
 # IBM Platform API Operator
 
+> **Important:** Do not install this operator directly. Only install this operator using the IBM Common Services Operator. For more information about installing this operator and other Common Services operators, see [Installer documentation](http://ibm.biz/cpcs_opinstall). If you are using this operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak to learn more about how to install and use the operator service. For more information about IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
+
 Operator used to manage IBM Platform API service. The Platform API provides a REST service for viewing information about your cluster, manage your cluster, install Helm charts, download CLI tools, and much more.
 
 ## Supported platforms
```

Comment:
I think the new requirement for supported platform is now "Red Hat OpenShift Container Platform 4.3 and above".
Note the Plataform was spelled wrong